.travis.yml

```diff
@@ -36,10 +40,11 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g npm@4
+  - npm install -g npm@5.7.1
```

Comment:
If you are not using `yarn` you should be using npm@5.7.1

Reply:
I think that is the case for apps, but not addons. I forget the exact reasoning. Seems safer to just leave that version at whatever the default `.travis.yml` has, since that works.

Reply:
I could switch npm back to v4 if you want, but your mocha test will not run using node 4 due to certain syntax.

Reply:
ahh. ok let's bump to 6 lol

Reply:
so for this, can we not add it, assuming it passes?

Reply:
Sure, I'll switch it back to npm@4

Reply:
Moved back to npm@4. Build passes.